### PR TITLE
Drawer fixes

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Fix editor height and 
- Add icon to button
- Convert json to yaml

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/components/ApiRules/ApiRuleDetails/ApiRuleDetails.js [hunk 1]
```diff
@@ -130,6 +132,7 @@ function ApiRuleDetailsHeader({ apiRule, setDrawerContent }) {
             {apiRule.json && (
               <Button
                 option="light"
+                glyph="syntax"
```
Comment (dariadomagala) [inline core-ui/src/components/ApiRules/ApiRuleDetails/ApiRuleDetails.js:135]: I'm not fully convinced that this icon is the right one, on a smaller screen it looks like two arrows

Comment (dariadomagala) [reply core-ui/src/components/ApiRules/ApiRuleDetails/ApiRuleDetails.js:135]: maybe something more basic like `document-text`?

Comment (akucharska) [reply core-ui/src/components/ApiRules/ApiRuleDetails/ApiRuleDetails.js:135]: I propose `attachment-html`

Comment (dariadomagala) [reply core-ui/src/components/ApiRules/ApiRuleDetails/ApiRuleDetails.js:135]: looks good :)

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**